binder/src/main/java/io/grpc/binder/BinderServerBuilder.java

```diff
@@ -48,7 +47,6 @@
 /**
  * Builder for a server that services requests from an Android Service.
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/8022")
 public final class BinderServerBuilder
     extends ForwardingServerBuilder<BinderServerBuilder> {
 
```

Comment:
Before we do this, we should fix b/236232634 (easy to do).

Reply:
Resolved as part of [this commit](https://github.com/grpc/grpc-java/pull/9669/commits/095259b927532a828db651649a51a99988b2a360).